src/rust-crypto/rust-crypto.ts

```diff
@@ -428,6 +432,46 @@ export class RustCrypto implements CryptoBackend {
         };
     }
 
+    /**
+     * Implementation of {@link CryptoApi#bootstrapSecretStorage}
+     */
+    public async bootstrapSecretStorage({
+        createSecretStorageKey,
+        setupNewSecretStorage,
+    }: ICreateSecretStorageOpts = {}): Promise<void> {
+        // If setupNewSecretStorage is not set, we stop
+        if (!setupNewSecretStorage || !createSecretStorageKey) return;
```

Comment:
This is inconsistent with the documentation on `setupNewSecretStorage`:

> Reset even if keys already exist.

We shouldn't return unless:

 * we already have an AES key, *AND*
 * `setupNewSecretStorage` is false.